### PR TITLE
Add prevalidation jobs to ironic-agent-image for OCP 4.10

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by andfasano** [comment]
@elfosardo looks like `.ci-operator` is [missing](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/25514/rehearse-25514-pull-ci-openshift-ironic-agent-image-release-4.10-prevalidation-e2e-metal-ipi-ovn-dualstack-prevalidation/1484478417507192832/build-log.txt) from the repo?

```
.ci-operator.yaml file: open .ci-operator.yaml: no such file or directory
```

**Comment by elfosardo** [comment]
> @elfosardo looks like `.ci-operator` is [missing](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_release/25514/rehearse-25514-pull-ci-openshift-ironic-agent-image-release-4.10-prevalidation-e2e-metal-ipi-ovn-dualstack-prevalidation/1484478417507192832/build-log.txt) from the repo?
> 
> ```
> .ci-operator.yaml file: open .ci-operator.yaml: no such file or directory
> ```

it is indeed....

**Comment by elfosardo** [comment]
@andfasano https://github.com/openshift/ironic-agent-image/pull/34

File: ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.10__prevalidation.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-metal-ipi-prevalidation
+  run_if_changed: .*packages-list.*
+  steps:
+    cluster_profile: packet
```
Comment (andfasano) [inline ci-operator/config/openshift/ironic-agent-image/openshift-ironic-agent-image-release-4.10__prevalidation.yaml:50]: Please change all the cluster_profile to `equinix-ocp-metal`

**Pull request merged**